examples/train_from_config.py

```diff
@@ -0,0 +1,22 @@
+import plac
+from pathlib import Path
+from thinc.api import use_pytorch_for_gpu_memory
+from spacy_transformers import install_extensions, TransformerModelByName
+from spacy_transformers import Transformer
+import spacy_transformers.tok2vec
+from spacy.cli.train_from_config import train_from_config
+
+
+def main(config_path, train_path, eval_path, gpu_id):
```

Comment:
I know this is just a temp file, but we don't need the `gpu_id` parameter here: spaCy's `train_from_config` doesn't read it from the command line but from the config.